spec/board_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rspec'
+require '../lib/board.rb'
```

Comment:
**[OPTIONAL]**
* Let's use `require_relative` instead of `require` in order to run the tests from the root directory.

Reply:
Done!